### PR TITLE
CORE: Standardize get_epoch

### PR DESCRIPTION
**Pull request by coachchucksol**
+ Removed `unwraps` in slot_toggle
+ added and standardized usage of `get_epoch` helper function

### CODE REVIEW

File: vault_core/src/config.rs [hunk 1]
```diff
@@ -90,9 +91,8 @@ impl Config {
         }
     }
 
-    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, VaultError> {
-        slot.checked_div(self.epoch_length())
-            .ok_or(VaultError::InvalidEpochLength)
+    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, CoreError> {
```
Comment (buffalu) [inline vault_core/src/config.rs:94]: remove this function and use get_epoch everywhere, no need for one line wrapper fn :)

File: restaking_core/src/config.rs [hunk 2]
```diff
@@ -49,9 +50,8 @@ impl Config {
         }
     }
 
-    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, RestakingError> {
-        slot.checked_div(self.epoch_length())
-            .ok_or(RestakingError::InvalidEpochLength)
+    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, CoreError> {
```
Comment (buffalu) [inline restaking_core/src/config.rs:53]: same here, remove this fn + use get_epoch everywhere

File: vault_core/src/config.rs (hunk 1 above)
Comment (coachchucksol) [reply vault_core/src/config.rs:94]: Done!

File: restaking_core/src/config.rs (hunk 2 above)
Comment (coachchucksol) [reply restaking_core/src/config.rs:53]: Done

**Review by buffalu — APPROVED** [review]

**Review by ebatsell — APPROVED** [review]

**Pull request merged**